### PR TITLE
attendance ICFP and PLDI

### PR DESCRIPTION
**Pull request by xanovsky**

### CODE REVIEW

File: Cares/index.md [hunk 1]
```diff
@@ -75,6 +75,15 @@ CARES committee members may also attend other workshops and conferences sponsore
     <th><b>Conference/Meeting</b></th>
     <th><b>CARES member attendees</b></th>
  </tr>
+   </tr>
```
Comment (sampsyo) [inline Cares/index.md:78]: I think this closing tag is unbalanced? Should be fine to just delete this line.

**Review by sampsyo — CHANGES_REQUESTED** [review]
Thanks! Looks good except for one stray line that snuck in.

File: Cares/index.md (hunk 1 above)
Comment (xanovsky) [reply Cares/index.md]: will do!

**Review by xanovsky — COMMENTED** [review]
fixed unbalanced tag

**Comment by sampsyo** [comment]
Thank you!!

**Pull request merged**